package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fbemitter",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
I personally don't like bumping the version like this until a release is final. In react I always go to `-alpha` as the first thing before working on the next version (the beta, rc as needed) and remove the `-whatever` right before releasing.